cmd/watt/aggregator.go

```diff
@@ -17,7 +17,7 @@ import (
 
 type WatchHook func(p *supervisor.Process, snapshot string) WatchSet
 
-type aggregator struct {
+type Aggregator struct {
```

Comment:
Is this changed to uppercase for a reason? (I think in go that means it's now public, but I don't see any public usage of it.)

Reply:
It does mean it's now public, and it's public because `kubebootstrap` needs it now (to update the `requiredTypes` field).

Reply:
(cf `kubewatchman.go:109`)